browserexport/browsers/firefox.py

```diff
@@ -47,23 +48,23 @@ def extract_visits(cls, path: PathIshOrConn) -> Iterator[Visit]:
             )
 
     @classmethod
-    def data_directory(cls) -> Path:
+    def data_directory(cls) -> Paths:
         return handle_path(
             {
-                "linux": "~/.mozilla/firefox/",
+                "linux": ("~/.mozilla/firefox/", "~/snap/firefox/common/.mozilla/firefox/"),
                 "darwin": "~/Library/Application Support/Firefox/Profiles/",
             },
             browser_name=cls.__name__,
         )
 
     @classmethod
     def locate_database(cls, profile: str = "*") -> Path:
-        dd: Path = cls.data_directory()
+        dd = cls.data_directory()
```

Comment:
don't think type annotation is necessary here, mypy can infer it? but happy to put it back if you prefer

Reply:
yeah, free to remove it -- I think I just had it in one of the functions and copy/pasted it as I was creating new browser implementations